cmd/test.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018 Google Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/GoogleCloudPlatform/container-structure-test/pkg/drivers"
+	"github.com/GoogleCloudPlatform/container-structure-test/pkg/output"
+	"github.com/GoogleCloudPlatform/container-structure-test/pkg/types"
+	"github.com/GoogleCloudPlatform/container-structure-test/pkg/types/unversioned"
+	"github.com/GoogleCloudPlatform/container-structure-test/pkg/utils"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+var totalTests int
+
+var driverImpl func(drivers.DriverConfig) (drivers.Driver, error)
+var args *drivers.DriverConfig
+
+var TestCmd = &cobra.Command{
+	Use:   "test",
+	Short: "Runs the tests",
+	Long:  `Runs the tests`,
+	Args: func(cmd *cobra.Command, _ []string) error {
+		return validateArgs()
+	},
+	Run: func(cmd *cobra.Command, _ []string) {
+		testResults := Run()
+		out := &output.OutWriter{
+			// Format: "raw",
```

Comment:
What's this?

Reply:
Ah, I was originally planning on having two built-in formats for output, `raw` and `JSON` or something similar, that can be switched with a flag. I'll remove this for now until I implement it.